virtcontainers/network.go

```diff
@@ -832,6 +836,10 @@ func doNetNS(netNSPath string, cb func(ns.NetNS) error) error {
 	}
 	defer currentNS.Close()
 
+	if netNSPath == "" {
```

Comment:
Please add a comment here to make sure we don't miss the fact that if `netNSPath` is empty, the callback will be run in the current network namespace.

Reply:
Done